candle-nn/src/conv.rs

```diff
@@ -115,6 +116,21 @@ impl Conv2d {
     pub fn bias(&self) -> Option<&Tensor> {
         self.bias.as_ref()
     }
+
+    pub fn absorb_bn(&self, bn: &BatchNorm) -> Result<Self> {
+        let (w_bn, b_bn) = bn.weight_and_bias().unwrap();
```

Comment:
This should be an error rather than a unwrap, just use `candle::bail!("batch norm does not have weight_and_bias")` or something similar.

Reply:
Fixed.